ui/ui-components/jest.setup.js

```diff
@@ -1,8 +0,0 @@
-import { configure } from "enzyme";
-import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
-
-configure({ adapter: new Adapter() });
-
-if (!process.env.SERVER_TOKEN) {
-  process.env.SERVER_TOKEN = "test-server-token";
```

Comment:
confirming that this doesn't need to be set

Reply:
👍 Yep, as we aren't booting up the server to test this lib, as it's just react components.